onebusaway-android/src/main/java/org/onebusaway/android/report/ui/dialog/CustomerServiceDialog.java

```diff
@@ -123,11 +123,14 @@ protected void initView(View view, ObaAgencyWithCoverage coverage) {
                     public void onClick(View view) {
                         String locationString = getActivity().getIntent().
                                 getStringExtra(BaseReportActivity.LOCATION_STRING);
-                        String agencyName = "\nAgency: " + agency.getName();
-                        UIUtils.sendEmail(getActivity(), agency.getEmail(), locationString + agencyName);
+                        UIUtils.sendEmail(getActivity(), agency.getEmail(), locationString);
+
+                        String gaEventLabel = locationString != null ?
```

Comment:
Could we change the logic on this, where `analytics_label_customer_service_email` is always reported, and then a separate event ` analytics_label_customer_service_email_without_location` is only reported if the location is null?